src/api/spec/models/report_spec.rb

```diff
@@ -20,4 +20,14 @@
       it { expect(new_report.valid?).to be(true) }
     end
   end
+
+  describe '#other_reports_from_reportable' do
+    let(:comment) { report.reportable }
+    let(:report) { create(:report) }
+    let(:another_report) { create(:report, reportable: comment) }
+
+    before { another_report }
```

Comment:
What about `let!(:another_report) ...` instead of calling it in the `before` block?